app/src/notifications/notificationsActions.js

```diff
@@ -0,0 +1,19 @@
+export const SET_NOTIFICATION = 'SET_NOTIFICATION';
+
+export function resetNotification() {
```

Comment:
Any reason not to use a more modern form?
```
export const resetNotification = () => ({
    type: SET_NOTIFICATION,
    payload: {
      visible: false,
      content: '',
      type: 'notification'
    }
  });
```

Reply:
Nothing special... sorry as I always forget it! 
Already updated